### PR TITLE
quick fix for SDK instructions

### PR DESCRIPTION
**Pull request by animeshsingh**
Takes some excerpts from Christian's website PR. 

I will be revisiting after release to figure out the overlap with User guide.

### CODE REVIEW

File: sdk/README.md [hunk 1]
```diff
@@ -26,23 +26,31 @@ backed by Tekton.
 <!-- END of ToC generated by running ./tools/mdtoc.sh sdk/README.md -->
 
 
-## Extensions to the Kubeflow Pipelines SDK
+## Kubeflow Pipelines with Tekton SDK Overview
```
Comment (ckadner) [inline sdk/README.md:29]: You need to update the link in the table of contents above:

```Diff
<   - [Extensions to the Kubeflow Pipelines SDK](#extensions-to-the-kubeflow-pipelines-sdk)
---
>   - [Kubeflow Pipelines with Tekton SDK Overview](#kubeflow-pipelines-with-tekton-sdk-overview)
```

**Review by ckadner — CHANGES_REQUESTED** [review]

File: sdk/README.md (hunk 1 above)
Comment (ckadner) [reply sdk/README.md:29]: I would rename it to "SDK Packages" -- less wordy

**Review by ckadner — APPROVED** [review]
/lgtm

File: sdk/README.md [hunk 2]
```diff
@@ -144,8 +152,16 @@ export DEFAULT_STORAGE_SIZE=2Gi
...
 After compiling the `sdk/python/tests/compiler/testdata/parallel_join.py` DSL script
-in the step above, we need to deploy the generated Tekton YAML to our Kubernetes
-cluster with `kubectl`. The Tekton server will automatically start a pipeline run.
+in the step above, we need to deploy the generated Tekton YAML to Kubeflow Pipeline engine.
+
+You can run the pipeline directly using a pre-compiled file and KFP-Tekton SDK. For more details, please look at the [KFP-Tekton user guide SDK documentation](https://github.com/kubeflow/kfp-tekton/tree/master/guides/kfp-user-guide#2-run-pipelines-using-the-kfp_tektontektonclient-in-python)
+
+```python
+experiment = kfp_tekton.TektonClient.create_experiment(name=EXPERIMENT_NAME, namespace=KUBEFLOW_PROFILE_NAME)
+run = client.run_pipeline(experiment.id, 'parallal-join-pipeline', 'pipeline.yaml')
```
Comment (Udiknedormin) [inline sdk/README.md:161]: *'parallel-join-pipeline'?

**Pull request merged**